apps/ui/src/common/pages/lev-help.tsx

```diff
@@ -0,0 +1,18 @@
+import { FC, Fragment, createElement as h } from 'react';
```

Comment:
as above - Part of another (unapproved) PR. I don't think this should be included.